test/ci/deprovision.yml

```diff
@@ -19,7 +19,7 @@
       ec2:
         instance_ids: "{{ item.instance_id }}"
         region: "{{ aws_region }}"
-        state: stopped
+        state: "{{ aws_terminate | ternary('absent', 'stopped') }}"
```

Comment:
We don't want to set state to absent here, we want to stop and then cleanup job (forget what it's called) will get instances that were renamed in the next task.

Reply:
Right, `aws_terminate` should be `false` for rh-dev account.

CI AWS profile doesn't have this bot (yet), if I understood @wking correctly there is a monthly cleanup of unowned instances. This account has permissions to terminate VMs so we should be using those if possible